package.json

```diff
@@ -46,7 +46,7 @@
     "@babel/eslint-parser": "^7",
     "@brightspace-ui/stylelint-config": "^0.8",
     "@brightspace-ui/testing": "^0.3",
-    "@open-wc/semantic-dom-diff": "^0.19.9",
+    "@open-wc/semantic-dom-diff": "^0.20.0",
```

Comment:
Do we have this specified to the patch number because the `0.x` releases are considered major?

Is this dependency only used for tests? If so, does green CI mean this is a safe bump?

Reply:
Yeah I lost track of this, but it looks like the breaking change was "Drop support for Node@14" so I'm pretty confident green CI = good to go. I dot think we could just be doing `^0.20` here